node/src/Commands.ts

```diff
@@ -128,11 +128,16 @@ export interface SetOptions {
     /**
      *  `onlyIfDoesNotExist` - Only set the key if it does not already exist.
      * Equivalent to `NX` in the Valkey API. `onlyIfExists` - Only set the key if
-     * it already exist. Equivalent to `EX` in the Valkey API. if `conditional` is
-     * not set the value will be set regardless of prior value existence. If value
-     * isn't set because of the condition, return null.
+     * it already exist. Equivalent to `EX` in the Valkey API. `onlyIfEqual` - Only set the
+     * key if the comparison value equals to the key value. Equivalent to `IFEQ` in the Valkey API.
+     * if `conditional` is not set the value will be set regardless of prior value existence.
+     * If value isn't set because of the condition, return null.
```

Comment:
I propose to add line breaks for better readability.
> `onlyIfDoesNotExist` - ...
> `onlyIfExists` - ...

And move last two lines to `set` docs.